lib/maglev.rb

```diff
@@ -62,6 +62,10 @@ def test?
       (defined?(Rails.env) && Rails.env.test?) || !!defined?(RSpec)
     end
 
+    def production?
+      env_name == 'production'
```

Comment:
Where does env_name come from? Why not `defined?(Rails.env) && Rails.env.production?`?

Reply:
Very similarly defined method: https://github.com/Dan-Nolan/mag-lev/blob/7a82b1481144feb73ad9f31a748e18bdce925d2e/lib/maglev.rb#L78